src/components/Text/Text.jsx

```diff
@@ -30,7 +30,7 @@ Text.propTypes = {
   /** It's the name of the input it refers to. <br><i>This is only used for `label`.</i> */
   htmlFor: PropTypes.string,
   /** It change the color of the text. <br><i>This is only used for `label`</i>. */
-  light: PropTypes.bool,
+  color: PropTypes.string,
```

Comment:
@federicoweber I changed this so we can customize the color.. is that ok with what you had in mind?